doc/vlan/switchport-mode-support/Switchport Mode and VLAN CLI Enhancement.md

```diff
@@ -0,0 +1,760 @@
+
+<img src="https://user-images.githubusercontent.com/61490193/236170194-2423f230-483a-42b7-936c-5939bf0e84ae.png" width="65%" height="30%"> 
+
+**8.** Change Mode from Access to Routed
+
+Ethernet0 is in Access mode, switching an access port to routed is not possible until it has an untagged member. Changing  mode from access to routed will show following error:
+
+<img src="https://user-images.githubusercontent.com/61490193/236168568-40e2c53b-d674-49b5-8885-faefac2283e5.png" width="65%" height="30%"> 
+
+
+**9.**  Change Mode from Access to Trunk
+
+Ethernet0 is in Access mode, switching an access to trunk mode is possible and its untagged member will be retained.  After changing mode from access to trunk, all functionalities of a trunk mode can be used/configred on Port
+
+<img src="https://user-images.githubusercontent.com/61490193/236169645-6911ab37-8384-4c4e-b9d1-5e6af634497d.png" width="65%" height="30%">
+
+We have given examples/usage for switchport mode configuration from routed to access on physical port. This works in the same way for PortChannels as they do for physical ports.
+
+#### Configuring Port & PortChannel from Routed to Trunk 
+
+* Physical Port Configuration
+
+In these examples, Ethernet4 will be configured as “Trunk” from “Routed” on Port.
+
+By default, all ports are in routed mode and have IP assigned. For Switchport configuration we have to remove IP assignment otherwise CLI will show following Error:
+
+**For Ethernet4**
+
+<img src="https://user-images.githubusercontent.com/61490193/236170350-be5d23a1-ba77-4a1e-8cfb-5b9b795c48a9.png" width="65%" height="30%">
+
+
+Following Steps will be taken to configure “Trunk” from “Routed” on a Port
+
+**1.** IP Removal on  Ethernet4
+
+```
+admin@sonic:~$ sudo config interface ip remove Ethernet4 10.0.0.2/31
+```
+
+**2.** Configure Ethernet4 as Trunk
+
+```
+admin@sonic:~$ sudo config switchport mode Trunk Ethernet4       
+```
+<img src="https://user-images.githubusercontent.com/61490193/236542261-682ebb20-07bd-4a34-ab9b-dc92bfc95bf0.png" width="65%" height="30%">
+**3.** Untagged Vlan Member Assignment on Trunk Port
+
+```
+admin@sonic:~$ sudo config vlan member add 3 -u Ethernet4          
+```
+
+<img src="https://user-images.githubusercontent.com/61490193/236294289-44cbceff-a3be-4460-a5c0-a8f219720466.png" width="65%" height="30%">
+
+**4.** Multiple Untagged Vlan Member Assignment on Trunk Port
+Ethernet4 is in Trunk mode, it can have only 1 untagged member. Configuring More than 1 untagged member on trunk Port will show following error:
+
+
+<img src="https://user-images.githubusercontent.com/61490193/236170542-587403ed-3cf0-465a-9e58-fff7c8130508.png" width="65%" height="30%">
+
+
+**5.** Tagged Vlan Member Assignment on Trunk Port
+
+Ethernet4 is in trunk mode, it can have multiple tagged members. Configuring tagged member on trunk Port will show following
+
```

Comment:
What if you attempt to add a tagged vlan to the port that is alerady part of the untagged vlan? what is its behavior?

Reply:
A Port in access mode can have only one untagged Vlan member. if a user tries to add a tagged vlan to such a port that already has an untagged vlan it will through an error. Please see the above section, **point 4,6.** under **"Configuring Port & PortChannel from Routed to Access"**  for a specific example of this behavior.